onebusaway-android/src/main/java/org/onebusaway/android/report/ui/dialog/CustomerServiceDialog.java

```diff
@@ -123,7 +125,19 @@ protected void initView(View view, ObaAgencyWithCoverage coverage) {
                     public void onClick(View view) {
                         String locationString = getActivity().getIntent().
                                 getStringExtra(BaseReportActivity.LOCATION_STRING);
-                        UIUtils.sendEmail(getActivity(), agency.getEmail(), locationString);
+                        String obaRegionName = RegionUtils.getObaRegionName();
```

Comment:
Looks like a lot of this code to get the parameters for `UIUtils.sendEmail()` is duplicated in `ReportTypeListFragment.onItemClick()` - could you move some of this to another utility method that then calls `UIUtils.sendEmail()`?